Fabulous.CodeGen/src/Fabulous.CodeGen/Generator/CodeGenerator.fs

```diff
@@ -243,6 +244,7 @@ module CodeGenerator =
                     match m.Name with
                     | "created" -> sprintf "%s?%s: (%s -> unit)" commaSpace m.Name data.FullName
                     | "ref" ->     sprintf "%s?%s: ViewRef<%s>" commaSpace m.Name data.FullName
+                    | "key" ->     sprintf "%s?%s: %s" commaSpace m.Name m.InputType 
```

Comment:
There's a bunch of other checks like those 2 in the same file that should also have a `key` case.

Reply:
Ok